src/components/data-management/ProjectDetails.jsx

```diff
@@ -104,6 +115,47 @@ const ProjectDetails = ({ width, height }) => {
     }
   }, [samples, activeProject]);
 
+  const downloadData = async (type) => {
+    const experimentId = activeProject?.experiments[0];
+
+    try {
+      if (!environment) throw new Error('Environment is not set');
```

Comment:
why is this needed? the variable is not being used in the method

Reply:
Oh yea, it's not used anymore. I'll remove it,